README.md

```diff
@@ -138,16 +138,17 @@ You may find some Monokai color schemes for mintty to match Cmder [here](https:/
 
 ##### Command Line Arguments for `init.bat`
 
-| Argument                      | Description                                                                                      | Default                               |
-| ----------------------------- | ----------------------------------------------------------------------------------------------   | ------------------------------------- |
+| Argument                        | Description                                                                                      | Default                               |
+| -----------------------------   | ----------------------------------------------------------------------------------------------   | ------------------------------------- |
 | `/c [user cmder root]`          | Enables user bin and config folders for 'Cmder as admin' sessions due to non-shared environment. | not set                               |
 | `/d`                            | Enables debug output.                                                                            | not set                               |
 | `/git_install_root [file path]` | User specified Git installation root path.                                                       | `%CMDER_ROOT%\vendor\Git-for-Windows` |
 | `/home [home folder]`           | User specified folder path to set `%HOME%` environment variable.                                 | `%userprofile%`                       |
 | `/max_depth [1-5]`              | Define max recurse depth when adding to the path for `%cmder_root%\bin` and `%cmder_user_bin%`   | 1                                     |
-| `/svn_ssh [path to ssh.exe]`    | Define `%SVN_SSH%` so we can use git svn with ssh svn repositories.                                | `%GIT_INSTALL_ROOT%\bin\ssh.exe`      |
+| `/svn_ssh [path to ssh.exe]`    | Define `%SVN_SSH%` so we can use git svn with ssh svn repositories.                              | `%GIT_INSTALL_ROOT%\bin\ssh.exe`      |
 | `/user_aliases [file path]`     | File path pointing to user aliases.                                                              | `%CMDER_ROOT%\config\user-liases.cmd` |
 | `/v`                            | Enables verbose output.                                                                          | not set                               |
+| (custom arguments)              | User defined arguments processed by `flag_exists`. Type `%flag_exists% /?` for more useage.      | not set                               |
```

Comment:
I'm not sure how the launcher (`Cmder.exe`) passes arguments to `init.bat`. Are you also launching Cmder with custom flags like this: `Cmder.exe /noAutorun` or just `vendor\init.bat /noAutorun`?

Reply:
I use `vendor\init.bat /noAutorun`.
it is actually Conemu called init.bat **with arguments which have nothing to do with custom flags**.
This section in README.md is **Command Line Arguments for `init.bat`**. So it **has nothing to do** with the launcher, that's why I put it here instead of `Cmder.exe Command Line Arguments`.

Reply:
The way I understand this functionality is it is to pass args to init.bat via different task entries in the conemu gui it has nothing to do with the cmder.exe launcher.